src/renderer/App.css

```
@@ -26,8 +26,8 @@
 
 .flexBox {
   display: block;
-  user-select: all;
-  -webkit-user-select: all;
+  user-select: unset;
+  --webkit-user-select: unset;
```

Is there any typo in prefix?
```suggestion
  -webkit-user-select: unset;
```

---

Yep! I'm so sorry, I'm not getting good feedback from the linter or my IDE...